### PR TITLE
Bump FastSpar to 0.0.9

### PR DESCRIPTION
**Pull request by apcamargo**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by apcamargo** [comment]
@bioconda/core Does anyone know how to fix that? (`configure: error: could not find the library gfortran`)

I though that using `{{ compiler('fortran') }}` was enough to include the Fortran compiler in the build process.

**Comment by epruesse** [comment]
See also #12392

File: recipes/fastspar/meta.yaml [hunk 1]
```diff
@@ -41,17 +44,10 @@ about:
...
   license_file: LICENSE
-  summary: Rapid and scalable correlation estimation for compositional data
-  description: |
-    ``FastSpar`` is a C++ implementation of the SparCC algorithm
-    which is up to several thousand times faster than the original
-    Python2 release and uses much less memory. The ``FastSpar``
-    implementation provides threading support and a *p*-value
-    estimator which accounts for the possibility of repetitious data
-    permutations.
+  summary: "Rapid and scalable correlation estimation for compositional data"
```
Comment (epruesse) [inline recipes/fastspar/meta.yaml]: Why did you delete the description?

Comment (apcamargo) [reply recipes/fastspar/meta.yaml]: I don't know why I deleted this. I'll put it back right now.

**Comment by apcamargo** [comment]
@epruesse FastSpart seems to be building right now! Do you want to finish the work in your PR? I see that you still have to update the version (0.0.7 -> 0.0.9) and fix the armadillo version bug.

**Comment by apcamargo** [comment]
As https://github.com/bioconda/bioconda-recipes/pull/13102 was merged, this recipe builds correctly.

@epruesse What do you think? I already added a constraint to the `armadillo` version (`>=6.7,<=8.200.0`).

**Comment by epruesse** [comment]
LGTM. Let's have it :)
